src/App.vue

```diff
@@ -4,11 +4,12 @@
     <v-main>
       <RouterView />
     </v-main>
-    <TheFooter v-if="showNav && isSmallDevice" :variant="variant" />
+    <TheFooter v-if="showNav && isSmallDevice && isAuthenticated" :variant="variant" />
```

Comment:
This conditional is kinda long, does it make sense to make this a computed value?